mon-pix/translations/en.json

```diff
@@ -1187,6 +1187,10 @@
       },
       "already-shared": "Thank you, your results have been submitted!",
       "archived": "This customised test has been archived by the organiser.'<br>' It is no longer possible to submit results.",
+      "badge-card": {
+        "acquired": "",
+        "not-acquired": ""
```

Comment:
Proposition : 
```suggestion
        "acquired": "Obtained",
        "not-acquired": "Not obtained"
```

Reply:
changer pour awarded / not awarded finalement